apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -557,30 +557,45 @@ class TrygdetidServiceImpl(
             "1",
         )
 
-    private fun hentOpplysninger(avdoed: Grunnlagsdata<JsonNode>): List<Opplysningsgrunnlag> {
+    private fun hentOpplysninger(
+        avdoed: Grunnlagsdata<JsonNode>,
+        behandlingId: UUID,
+    ): List<Opplysningsgrunnlag> {
+        val avodedDoedsdato = avdoed.hentDoedsdato()
+
+        val avdoededsDatoOpplysning =
+            avodedDoedsdato?.verdi?.let {
+                Opplysningsgrunnlag.ny(TrygdetidOpplysningType.DOEDSDATO, avodedDoedsdato.kilde, it)
+            } ?: run {
+                throw TrygdetidMaaHaAvdoedDoedsdatoException(behandlingId)
+            }
+
         val foedselsdato = avdoed.hentFoedselsdato()
+        val foedselsdatVerdi =
+            foedselsdato?.verdi ?: run {
```

Comment:
Gir det noen fordeler ved å wrappe dette i en `run`? I stedet for f. eks
```kotlin
val foedselsdatoVerdi = avdoed.hentFoedselsdato()?.verdi ?: throw TrygdetidMaaHaFoedselsdatoException(behandlingId)

Reply:
Ah nei her gir det ingen fordeler. Fjerner.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3048/commits/0f843632e0aa6a97ef1a746240882b203d88e9d9